micro-rdk-server/esp32/main.rs

```diff
@@ -70,30 +72,38 @@ mod esp32 {
 
         let mut registry = Box::<ComponentRegistry>::default();
         register_example_modules(&mut registry);
+
         #[cfg(feature = "qemu")]
         let storage = { RAMStorage::new() }; //NVSStorage::new("nvs").unwrap();
         #[cfg(not(feature = "qemu"))]
         let storage = { NVSStorage::new("nvs").unwrap() };
+
         // At runtime, if the program does not detect credentials or configs in storage,
         // it will try to load statically compiled values.
 
         if !storage.has_wifi_credentials() {
+            log::warn!("no wifi credentials were found in storage");
+
             // check if any were statically compiled
             if SSID.is_some() && PASS.is_some() {
-                log::info!("Storing static values from build time wifi configuration to NVS");
+                log::info!("storing static values from build time wifi configuration to storage");
```

Comment:
I've tried to consistently move to lowercase initial word in log messages I added or were nearby my changes.